design/authenticators/authn_azure/authn_azure_solution_design.md

```diff
@@ -552,7 +552,7 @@ The default TTL of an Azure access token is one hour, but it can reduced to a sh
 - [ ] Security action items were taken care of
 - [ ] Performance tests were done and align with SLA
 - [ ] Logs were reviewed by TW and PO
-- [ ] Documentation has been written and reviewed by PO and TW for both DAP and Conjur
+- [ ] Documentation has been written and reviewed by PO and TW for both Conjur Enterprise and Conjur
```

Comment:
should this be `both Conjur Enterprise and Conjur Open Source`? if so - can you look for other occurrences?

Reply:
DAP should be changed to conjur enterprise this is this rename

Reply:
@hilagross fix me if im wrong

Reply:
actually I'm not sure we should change design pages, I think we should change only readme files

Reply:
the effort is minimal so i would change it. i agree that it's not very important.

Reply:
> should this be both Conjur Enterprise and Conjur Open Source? if so - can you look for other occurrences?

@tzheleznyak i was referrring to the fact that you left `Conjur` as is and didn't change it to `Conjur Open Source`